src/js/fluxcontroller.js

```diff
@@ -610,7 +610,7 @@ define(function (require, exports, module) {
             .bind(this)
             .then(function () {
                 var actionPromise = action.apply(actionReceiver, params);
-                if (!(actionPromise instanceof Promise)) {
+                if (!(actionPromise.then)) {
```

Comment:
This is not easily fixable with a polyfill, but `bluebird.Promise` is confused by browser's `Promise`, so it's better to check for presence of `then` function (and confirm it's a thenable)